### PR TITLE
Hosted Private Cloud : Guides for IAM activation (faq, getting started, how to)

### PR DESCRIPTION
**Pull request by pg-ovh**
I added just few details balises, just remove it if it's to tricky for you guys to have it. It for the schema only, to reduce page size.

### CODE REVIEW

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: "Guides premiers pas avec IAM dans mon cloud privée VMware managé par OVHcloud"
+excerpt: "Comment activer avec IAM dans une infrastructure VMware managé par OVHcloud"
+updated: 2024-05-24
+---
+
+# Table des matières
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]: This index is not necessary. We only add it on very long guides. Please remove it.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+5. [Aller plus loin](#Aller plus loin)
+
+---
+## Objectif
+**Ce guide vous détaille comment activer IAM dans un Cloud Privée VMware par OVHcloud**
+
+---
+## Prérequis
+
+- Avoir un compte OVHcloud (voir guide :[Comment créer un compte OVHcloud](https://help.ovhcloud.com/csm/fr-account-create-ovhcloud-account?id=kb_article_view&sysparm_article=KB0043023)).
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]: Use the valid links formats `[link text](/pages/my-guide-folder)`for all links

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+---
+## Instruction
+
+## Activer IAM dans un PCC
+> [!CAUTION]
+> Cette opération peut prendre jusqu’à 30 minutes.
+
+### Pour activer IAM via l'espace client :
+
+1. Accéder à la console OVHcloud, en suivant [le lien de l'espace client](https://www.ovh.com/manager) et Connectez vous avec vos identifiants.
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]:
```suggestion
1. Accéder à la console OVHcloud, en suivant [le lien de l'espace client](/links/manager) et Connectez vous avec vos identifiants.
```

Please duplicate this fix for all other manager links

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+- Avoir un compte OVHcloud (voir guide :[Comment créer un compte OVHcloud](https://help.ovhcloud.com/csm/fr-account-create-ovhcloud-account?id=kb_article_view&sysparm_article=KB0043023)).
+- Disposer au préalable d'un ou plusieurs produits liés à ce compte OVHcloud (Hosted Private Cloud powered by VMware, Service Pack VMware etc..)
+- Savoir ["Créer et gérer des utilisateurs locaux sur un compte OVHcloud"](https://help.ovhcloud.com/csm/fr-account-managing-users?id=kb_article_view&sysparm_article=KB0043058).
+- Savoir ["Comment utiliser les politiques IAM depuis votre espace client"](https://help.ovhcloud.com/csm/fr-customer-iam-policies-ui?id=kb_article_view&sysparm_article=KB0058730).
+
+---
+## Instruction
+
+## Activer IAM dans un PCC
+> [!CAUTION]
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]:
```suggestion
> [!!warning]
```
Please duplicate this fix if needed

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+---
+
+# Table des matières
+1. [Objectif](#Objectif)
+2. [Prérequis pour commencer](#Prérequis)
+3. [Instruction - Activer IAM](#Instruction)
+4. [A suivre - Comment créer un rôle IAM Vsphere](#Asuivre)
+5. [Aller plus loin](#Aller plus loin)
+
+---
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]:
```suggestion

```
Please remove all those lines

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+title: "Guides premiers pas avec IAM dans mon cloud privée VMware managé par OVHcloud"
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]: Too much translated.
The name of the offer is VMware on OVHcloud, even for France
see https://www.ovhcloud.com/fr/hosted-private-cloud/

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+---
+## Prérequis
+
+- Avoir un compte OVHcloud (voir guide :[Comment créer un compte OVHcloud](https://help.ovhcloud.com/csm/fr-account-create-ovhcloud-account?id=kb_article_view&sysparm_article=KB0043023)).
+- Disposer au préalable d'un ou plusieurs produits liés à ce compte OVHcloud (Hosted Private Cloud powered by VMware, Service Pack VMware etc..)
+- Savoir ["Créer et gérer des utilisateurs locaux sur un compte OVHcloud"](https://help.ovhcloud.com/csm/fr-account-managing-users?id=kb_article_view&sysparm_article=KB0043058).
+- Savoir ["Comment utiliser les politiques IAM depuis votre espace client"](https://help.ovhcloud.com/csm/fr-customer-iam-policies-ui?id=kb_article_view&sysparm_article=KB0058730).
+
+---
+## Instruction
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]:
```suggestion
## En pratique
```

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+
+## Activer IAM dans un PCC
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]:
```suggestion
### Activer IAM dans un PCC
```
Please increase hierarchy after the `## En pratique` title

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+
+![Activer IAM](images/iam_enable_3.png){.thumbnail }
+
+### Comment activer IAM dans un PCC depuis l'API OVHcloud ?
+
+> [!CAUTION]
+> Cette opération pourra prendre jusqu’à 30 minutes.
+
+### Pour activer IAM via l'API :
+> [!IMPORTANT]
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]: This won't work, use only the accepted markdown formats

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+>
+> @api {v1} GET /dedicatedCloud/{serviceName}/iam
+>
+
+> **Parameters:**
+>
+> serviceName: La référence de votre service pcc, `pcc-XX-XX-XX-XX`.
+
+RETOUR :
+```Shell
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]:
```suggestion
```shell
```
Please replicate the fix

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+  "message": "This option is already enabled"
+}
+```
+
+---
+## A suivre  
+
+### Créer un rôle IAM Vsphere
+
+Je vous invite maintenant à suivre le guide 3 : [Comment créer un rôle IAM](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role)
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]: Not the right tone, please prefer "Vous pouvez maintenant suivre..."

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+
+RETOUR :
+```Shell
+{
+  "message": "This option is already enabled"
+}
+```
+
+---
+## A suivre
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]: To remove

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+- Ressources : **PCC-XXX**
+- Actions : **Groupes de permissions, Ajoutées manuellement -> Rôle Vsphere/IAM**
+- Utilisateurs : **Utilisateur 1/2/3**
+
+<details>
+<summary>Schema IAM
+
+</summary>
+
+<summary> Voir le diagramme du fonctionnement de IAM avec un cloud privé OVHcloud :</summary>
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md]: check again the right `<details>` format on our internal pages.
This needs to be fixed and this format only used on rare occasions.
All the images links have also to be written in HTML if the `<details>` format is used.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md [hunk 14]
```diff
@@ -0,0 +1,138 @@
...
+Pour utiliser IAM dans votre cloud privé, il va faut : 
+
+- [x] Activer IAM.
+
+- [x] Créer une politique IAM globale liée à mon identité OVHcloud.
+
+Des étapes supplémentaires sont nécessaires, nous allons les détailler dans la suite de ce guide.
+
+### Comment fonctionne IAM dans un cloud privée VMware sur OVHcloud ?
+> [!IMPORTANT]
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md]: Not a valid format

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md [hunk 15]
```diff
@@ -0,0 +1,138 @@
...
+**Dans ce guide, nous allons vous expliquer comment démarrer avec IAM dans votre Cloud Privée VMware sur OVHcloud et voir quelques questions réponses dans la FAQ** 
+
+---
+## Etape 1 : En pratique
+
+## Premiers pas avec IAM et mon cloud privée VMware managé par OVHcloud
+
+Pour utiliser IAM dans votre cloud privé, il va faut : 
+
+- [x] Activer IAM.
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md]: Not a valid format

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md [hunk 16]
```diff
@@ -0,0 +1,138 @@
...
+
+![Schema IAM](images/iam_vmware_schema_2.png){.thumbnail}
+
+![Schema IAM](images/iam_vmware_schema_3.png){.thumbnail}
+
+</details>
+
+## Etape 2 : FAQ
+
+## Dans quelle scope applicable puis-je utiliser IAM au sein de mon cloud privé OVHcloud et quelles sont ses limitations ?
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md]: Markdown hierarchy is not good

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role/guide.fr-fr.md [hunk 17]
```diff
@@ -0,0 +1,119 @@
...
+> [!TIP]
+> Les rôles IAM sont préfixés par : {{iam-}}
+
+Accéder à la console OVHcloud, en suivant [le lien de l'espace client](https://www.ovh.com/manager) et **Connectez-vous avec vos identifiants**.
+
+Pour créer votre rôle IAM dans le PCC : 
+
+1. Allez dans la section de votre cloud privée -> `Hosted Private Cloud > PCC-XX > Utilisateurs`
+
+2. Cliquez sur -> `Créer un rôle IAM`
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role/guide.fr-fr.md]:
```suggestion
2. Cliquez sur `Créer un rôle IAM`{.action}
```
Please duplicate this fix for all similar buttons descriptions.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role_policy/guide.fr-fr.md [hunk 18]
```diff
@@ -0,0 +1,129 @@
...
+
+## Instruction
+
+## Etape 1
+
+### Créer une politique ? (optionnel)
+
+### Accéder à IAM 
+
+1. Pour accéder au service IAM, suivez les indications de la capture suivante : `SUPPORT STANDARD > IDENTITES ET ACCES (IAM)`{.action}
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role_policy/guide.fr-fr.md]: Too many numbers can lead to confusion.
Also, the text should contain all the necessary information. Screenshots are a + but shall not be considered as standalone explanations.
```suggestion
1Pour accéder au service IAM, suivez les indications de la capture suivante : `SUPPORT STANDARD > IDENTITES ET ACCES (IAM)`{.action}
```

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role_policy/guide.fr-fr.md [hunk 19]
```diff
@@ -0,0 +1,129 @@
...
+
+![IAM] OVHCLOUD(images/iam_role_policy_9.png){.thumbnail}
+
+2. Cliquer sur -> `CREER UNE POLITIQUE`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role_policy/guide.fr-fr.md]: Please use lowercase for buttons (just replicate them from their interface)

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]
This review is not exhaustive.
Please replicate all suggestions on all related formats which need fixes.

Some images contain some FR notes, they have to be in Full English.
Some images contain many arrows, which could be confusing. I suggest using less circling and arrows.

**Comment by pg-ovh** [comment]
Hello @OVH-Team-Guides I have added the necessary editions required, let me know if it's okay.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md (hunk 15 above)
Comment (pg-ovh) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md]: Not valid, this is pure Markdown ?

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 20]
```diff
@@ -1,78 +1,87 @@
 ---
-title: "Guides premiers pas avec IAM dans Hosted Private Cloud - VMware on OVHcloud"
-excerpt: "Découvrez comment activer IAM dans Vsphere"
-updated: 2024-05-17
+title: "IAM pour VMware on OVHcloud - Comment activer IAM"
+excerpt: "Découvrez comment activer IAM pour votre offre VMware on OVHcloud via l'espace client OVHcloud ou par le biais de l'API OVHcloud"
```
Comment (pg-ovh) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md:3]: On met dans le exerpt : "...via l'espace client OVHcloud ou par le biais de l'API OVHcloud"
Pour améliorer le referencement @Y0Coss, ça rallonge je trouve, mais si la taille n'a pas d'importance c'est mieux en effet.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md [hunk 21]
```diff
@@ -1,78 +1,87 @@
...
 ### Activer IAM
-> [!warning]
+
+> [!primary]
 > Cette opération peut prendre jusqu’à 30 minutes.
 
-### Via l'espace client :
+### Via l'espace client OVHcloud
 
-1. Accéder à la console OVHcloud, en suivant [le lien de l'espace client](/links/manager) et connecter vous avec vos identifiants.
```
Comment (pg-ovh) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]: Je préfere les numérotations car ça permet de vraiment voir les étapes importante et nécéssaire à la bonne éxécution du guide "tutoriel". Dans un guide plus sommaire en effet ça peut porter à confusion.
Exemple : https://cloud.ibm.com/docs/vmware-service?topic=vmware-service-vmware-as-a-service-vdc

**Review by pg-ovh — COMMENTED** [review]
I have added comments @Y0Coss, I would recommend waiting for Aymeric feedback before english translation. Your recommendations are perfect. I'll add them in the next guides. Thanks.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md (hunk 20 above)
Comment (OVH-Team-Guides) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md:3]: L'excerpt peut en effet être plus long, tu peux le tester dans https://www.highervisibility.com/seo/tools/serp-snippet-optimizer/ (champ "Description"). Ici il fait 835px sur 990px max conseillés, donc il est dans les clous.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md (hunk 21 above)
Comment (OVH-Team-Guides) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md]: Je comprends ton point de vue mais la numérotation n'est pas applicable pour ce type de descriptions. Tous les guides OVHcloud, à quelques exceptions près peut-être, suivent ce modèle.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role/guide.fr-fr.md [hunk 22]
```diff
@@ -30,7 +30,7 @@ Connectez-vous à votre [espace client OVHcloud](/links/manager) et cliquez sur
 
 Cliquez sur la rubrique `VMware`{.action}, sélectionnez votre infrastructure puis rendez-vous dans l'onglet `Utilisateurs`{.action}.
 
-Cliquez sur `Créer un rôle IAM`{.action}
+Cliquez sur `Créer un rôle IAM`{.action}.
 
 ![IAM role add](images/iam_role_8.png){.thumbnail}
```
Comment (pg-ovh) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role/guide.fr-fr.md:36]: Il faut verifier ça, car il semble que les permissions managés (actions auto) ne soit pas encore géré par Vsphere (uniquement les actions manuelles) : 
"Vous pouvez aussi faire le choix d'être **vSphereAdmin** dans les permissions managées de la politique IAM."

Comment (OVH-Team-Guides) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role/guide.fr-fr.md:36]: Fixed with the next commit

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @pg-ovh

**Pull request merged**